mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -2090,7 +2089,7 @@ func flattenSettings(settings *sqladmin.Settings, d *schema.ResourceData) []map[
 	data := map[string]interface{}{
 		"version":                     settings.SettingsVersion,
 		"tier":                        settings.Tier,
-		"edition":                     flattenEdition(settings.Edition),
+		"edition":                     settings.Edition,
```

Comment:
The CloudSQL API actually uses default Edition as "EDITION_UNSPECIFIED" (refer: type [Settings](https://pkg.go.dev/google.golang.org/api/sqladmin/v1beta4#Settings)) when the customer doesn't specify the edition. Today a "EDITION_UNSPECIFIED" instance is configured similar to a "ENTERPRISE" instance but this could change within the CloudSQL API which will create inconsistent behavior for the customers when dealing directly with CloudSQL API vs Terraform provider.